selfdrive/car/nissan/values.py

```diff
@@ -39,7 +39,7 @@ class NissanCarInfo(CarInfo):
 
 CAR_INFO: Dict[str, Optional[Union[NissanCarInfo, List[NissanCarInfo]]]] = {
   CAR.XTRAIL: NissanCarInfo("Nissan X-Trail 2017"),
-  CAR.LEAF: NissanCarInfo("Nissan Leaf 2018-22"),
+  CAR.LEAF: NissanCarInfo("Nissan Leaf 2018-22"), 
```

Comment:
```suggestion
  CAR.LEAF: NissanCarInfo("Nissan Leaf 2018-22", video_link="https://youtu.be/vaMbtAh_0cY"),
```

@cellfreak this is how you do it